File: Codeforces/C478B.java
import java.util.*;
public class Sample {

	public static void main(String[] args) {
		Scanner mohit = new Scanner(System.in);
		long n,m;
		n = mohit.nextLong();
		m = mohit.nextLong();
		long kmin,kmax;
		kmax = ((n-m+1)*(n-m))/2;
		kmin = (((n%m)*((n/m)+1)*(n/m))+(m- n%m)*(n/m)*((n/m)-1))/2;
		System.out.println(kmin+" "+kmax);
		mohit.close();
	}
}

File: Codeforces/C478B.py
n, m = map(int, input().split())
if((n-m)%2):
	kmax = int((n-m+1)/2)*(n-m)
else:
	kmax = int((n-m)/2)*(n-m+1)
k = int(n/m)
kmin = int(((n%m)*(k+1)*k)/2)+int(((m- n%m)*k*(k-1))/2)
print(kmin, kmax)